src/imitation/util/registry.py

```diff
@@ -102,22 +91,3 @@ def wrapper(path: str, venv: VecEnv) -> T:
         return fn(venv, **kwargs)
 
     return wrapper
-
-
-def dummy_context(fn: Callable[..., T]) -> Callable[..., ContextManager[T]]:
-    @functools.wraps(fn)
-    @contextlib.contextmanager
-    def wrapper(*args, **kwargs) -> Iterator[T]:
-        yield fn(*args, **kwargs)
-
-    return wrapper
-
-
-def sess_context(fn: Callable[..., T]) -> Callable[..., ContextManager[T]]:
```

Comment:
Just want to confirm that garbage collection definitely frees all the resources we need in PyTorch?

Reply:
Yes, assuming there are no reference cycles.

Reply:
(Torch programs can have memory leaks due to, e.g., keeping a hold of things that are part of the gradient graph, but unfortunately the only way to debug those things is to find those references and delete them)